airbyte-integrations/connectors/source-infinicept/integration_tests/sample_state.json

```diff
@@ -0,0 +1,5 @@
+{
+  "todo-stream-name": {
+    "todo-field-name": "value"
+  }
+}
```

Comment:
this is only used when you have incremental streams. not your case you can delete it